main.js

```diff
@@ -0,0 +1,66 @@
+var players = [];
+var markers = ["X", "O"];
+var gameOver = false;
+var scores = [0, 0];
+var winValues = [7, 56, 73, 84, 146, 273, 292, 448];
```

Comment:
From what I am seeing in your code, you are trying to identify win conditions by implementing specific values to each of the clickable div elements and comparing them to this array. Due to the complexity of your solution, you should write a comment in your code that describes the purpose of this array, as well as cellsArray, winChecker and playerFunction